ZuluContent/Engines/Harvest/Core/HarvestSystem.cs

```diff
@@ -188,7 +188,7 @@ public virtual bool BeginHarvesting(Mobile from, Item tool)
                     vein = def.GetColoredVein(from, tool, ref harvestAmount);
                     if (vein == null)
                     {
-                        def.SendMessageTo(from, "You fail to find any colored ore.");
+                        def.SendMessageTo(from, "You fail to find any colored veins.");
```

Comment:
Could conditionally set the name like

```csharp
var veinName = def.Skill == SkillName.Lumberjacking ? "logs" : "ores";
def.SendMessageTo(from, $"You fail to find any colored {veinName}.");
```

Reply:
yeah fair enough I admit was being lazy here!